auth.mkd

```diff
@@ -172,43 +172,43 @@ Valid Not Before (VNB) Timestamp by UA (4-bytes):
+The UA's HI SHOULD be retrieved from DNS. If not available it may have been revoked. Note that accurate revocation status is a DIME inquiry; DNS non-response is a hint that a DET is expired or revoked. It MAY be retrieved from a local cache, if present. The local cache is typically populated by DNS lookups and/or by received Broadcast Endorsements ({{dime-attestation}}).
 
 Once the Observer has the registered UA's DET and HI, all further (or cached previous) DRIP-based Authentication Messages using the UA DET can be validated. Signed content, tied to the DET, can now be trusted to have been signed by the holder of the private key corresponding to the DET.
 
-Whether the content is true is a separate question which DRIP cannot address but sanity checks ({{reqs}}) are possible.
+Whether the content is true is a separate question which DRIP cannot address, but sanity checks ({{reqs}}) are possible.
 
 ### DIME Endorsements of Subordinate DET {#dime-attestation}
 
-When an Observer receives a DRIP Link Authentication Message ({{drip-link}}) containing an Endorsement by the DIME of the a child DET registration, it SHOULD validate the signature using the HI corresponding to the DIME's DET.
+When an Observer receives a DRIP Link Authentication Message ({{drip-link}}) containing an Endorsement by the DIME of a child DET registration, it SHOULD validate the signature using the HI corresponding to the DIME's DET.
 
-The DIME's HI, SHOULD be retrieved from from DNS (Section 5, {{drip-registries}}), when available. It MAY be cached from a prior DNS lookup or it may be stored in a distinct local store.
+The DIME's HI, SHOULD be retrieved from DNS (Section 5, {{drip-registries}}), when available. It MAY be cached from a prior DNS lookup or it may be stored in a distinct local store.
 
 ### DIME Hierarchy Endorsements {#dime-hierarchy}
 
 An Observer can receive a series of DRIP Link Authentication Messages ({{drip-link}}), each one pertaining to a DIME's registration in the DIME above it in the hierarchy. Similar to {{dime-attestation}}, each link in this chain SHOULD be validated.
 
 ### UAS RID Trust {#rid-trust}
 
-{{ua-attestation}}, {{dime-attestation}}, and {{dime-hierarchy}} complete the trust chain but the chain cannot yet be trusted as having any relevance to the observed UA because replay attacks are trivial. At this point the key nominally possessed by the UA is trusted but the UA has not yet been proven to possess that private key.
+{{ua-attestation}}, {{dime-attestation}}, and {{dime-hierarchy}} complete the trust chain, but the chain cannot yet be trusted as having any relevance to the observed UA because replay attacks are trivial. At this point, the key nominally possessed by the UA is trusted but the UA has not yet been proven to possess that private key.
```

Comment:
"trust chain, but the chain cannot yet be trusted": a naïve reader may see this as an oxymoron

Reply:
```suggestion
{{ua-attestation}}, {{dime-attestation}}, and {{dime-hierarchy}} complete the trust chain for the claimed DET and associated HI (public key), but the chain cannot yet be trusted as having any relevance to the observed UA because replay attacks are trivial. At this point, the key nominally possessed by the UA is trusted but the UA has not yet been proven to possess that private key.
```

This is the suggestion from Stu, would this suffice?